manageiq-operator/controllers/manageiq_controller.go

```diff
@@ -166,6 +247,90 @@ func (r *ManageIQReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 var logger = log.Log.WithName("controller_manageiq")
 
+func FindDeployment(cr *miqv1alpha1.ManageIQ, client client.Client, name string) *appsv1.Deployment {
```

Comment:
In a followup, do you think we should lower case these methods? I think they can be private.